Assets/Prefabs/UI/StartSplashScreen/StartSplashScreen.controller

```diff
@@ -0,0 +1,72 @@
+%YAML 1.1
```

Comment:
Pls create Animationfolder in assets folder and move this file to created folder.